examples/config.hocon.sample

```diff
@@ -52,6 +56,39 @@ aws {
   secret-key: iam
 }
 
+# config for NSQ sink and NSQ source
+NSQ {
+  #Topic name for good event source
+  good-source-topic: "{{sinkKinesisNsqSourceGoodTopicName}}"
+
+  #Channel name for good event source
+  good-source-channel: "{{sinkKinesisNsqSourceGoodChannelName}}"
+
+  #Topic name for good event sink
+  good-sink: "{{sinkKinesisNsqSinkGoodTopicName}}"
+
+  #Topic name for bad event source
+  bad-source-topic: "{{sinkKinesisNsqSourceBadTopicName}}"
+
+  #Channel name for bad event source
+  bad-source-channel: "{{sinkKinesisNsqSourceBadChannelName}}"
+
+  #Topic name for bad event sink
+  bad-sink: "{{sinkKinesisNsqSinkBadTopicName}}"
+
+  #Topic name for plain JSON
+  plain-json-topic: "{{sinkKinesisNsqSinkPlainJsonTopicName}}"
+     
+  #Host name for NSQ tools
+  host: "{{NsqHost}}"
+
+  #Port for nsqd
+  port: "{{NsqdPort}}"
+
+  #Port for nsqlookupd
+  lookup-port: {{NsqlookupdPort}}
+}
```

Comment:
It'd be cool to incorporate this configuration to a single `source` configuration similar to what has been done for stream-enrich and `streams`